Makefile

```diff
@@ -1,6 +1,10 @@
 SHELL := /bin/bash
 ENABLE_FEATURES ?= default
 
+ifeq ($(TCMALLOC),1)
+ENABLE_FEATURES = tcmalloc
+endif
```

Comment:
The calls to cargo with `--disable-default-features` will build with the system allocator, and we need it to build with jemalloc. So there needs to be an else case here that adds the jemalloc feature.